app/_data/docs_nav_ee_2.1.x.yml

```diff
@@ -385,6 +385,8 @@
       url: /developer-portal/legacy-migration
     - text: Structure and File Types
       url: /developer-portal/structure-and-file-types
+    - text: Using the Portal Files API
+      url: /developer-portal/files-api
```

Comment:
Should we put this next to the Helpers CLI in the table of contents?

Reply:
> Should we put this next to the Helpers CLI in the table of contents?

I placed it near the topic that seemed most relevant about the file types. The former API was at the bottom of a files topic https://docs.konghq.com/enterprise/0.36-x/developer-portal/content-management/file-management/. I think it's fine where it is. @Jlawlzz ?